### ISSUE TITLE
Running the Script

### ISSUE THREAD

**Issue by rjvim** [dev-chauhan/pdfext#1]
Hi @dev-chauhan,

Would you please give me instructions on how to run the code, as in commands.

I will try to deploy and figure out what changes we need to do so that this runs on the server.

**Comment by dev-chauhan** [comment]
On shell you can run as below
```
python main.py "path of the file"
```
Extracted json will be created in directory named extracted which will be in the same directory.

**Comment by rjvim** [comment]
Hi @dev-chauhan I need to host this code in AWS Lambda, so I need one function which works like this:

```
handleFunction(url_of_pdf_file){

1. PDF file would be hosted somewhere and python script needs to download from url_of_pdf_file

2. Parse the PDF

3. return JSON (Don't write to file/directory)

}
```

I think the additional step would be fetching the file.

**Comment by dev-chauhan** [comment]
@rjvim 
implemented a `extract_from_pdf_url` in `extract.py` which you can use to achieve above functionality.

**Comment by rjvim** [comment]
Hi Dev,

Does it download a file for internet?

**Comment by dev-chauhan** [comment]
@rjvim 
No, it does not download permanently on disk, but stores in volatile memory after receiving pdf from URL. I thought this way it will be better, do you want to write pdf on disk?

**Comment by rjvim** [comment]
That is fine. So, if I give a URL it can manage it right? The pdf file might be hosted anywhere on internet like may be Dropbox with public access, so I will give URL to that file.

**Comment by dev-chauhan** [comment]
Yes
